.storybook/main.ts

```diff
@@ -7,6 +7,14 @@ const config: StorybookConfig = {
     '@storybook/addon-essentials',
     '@storybook/addon-onboarding',
     '@storybook/addon-interactions',
+    {
```

Comment:
👍  this is as nice addition. 

I was adding some basic docs via JSDOCs tags but this is better to allow us to add better documentation